_includes/footer-cta.html

```diff
@@ -0,0 +1,23 @@
+<div class="footer-cta outer-section-padding">
+  <div class="container">
+    <div class="row">
+      <div class="col-12 col-lg-6">
+        <div class="brand-list text-center text-lg-left">
```

Comment:
can we make brand-list an include with parameters? Well, it's not used in other parts of the site tho